bears/coffee_script/CoffeeLintBear.py

```diff
@@ -239,6 +241,9 @@ class Foo
         :param allow_empty_functions:
             Allows declaring empty functions.
         """
+        if not max_line_length:
```

Comment:
`max_line_length == 0` is explicit, use that

Reply:
question: what happens when you pass 0 to `coffeelint`? Instead of us changing it to `sys.maxsize` and then passing that value, will `coffeelint` automatically recognise `max_line_length=0` as infinite length?

Reply:
Yeah @adtac 
Indeed coffeelint automatically detects 0 as some large length, but I couldn't find that documented properly in the docs, although it just works.
So should we depend on this behaviour ?

Reply:
Yes, let's use the built-in behaviour instead of coming up with a fix on our side. So no changes to the code, but the new test is useful.

Reply:
Sure,
I will just modify the PR accordingly.
Thanks :smile:

Reply:
@adtac 
Changed the PR to just the test addition.
Please review when possible :smiley: